### ISSUE TITLE
Visual Studio Code Port?

### ISSUE THREAD

**Issue by zaynali53** [luisfer0793/CarbonTheme#4]
Love this theme. Thank you so much.

Any further plans?

**Comment by Feel-ix-343** [comment]
I actually just found this https://marketplace.visualstudio.com/items?itemName=MaxAndyAng.vscode-carbon-theme

**Comment by mwamodo** [comment]
> I actually just found this https://marketplace.visualstudio.com/items?itemName=MaxAndyAng.vscode-carbon-theme

@Feel-ix-343 Thanks. Its a good option

**Comment by juanmsmilla** [comment]
> > I actually just found this https://marketplace.visualstudio.com/items?itemName=MaxAndyAng.vscode-carbon-theme
> 
> @Feel-ix-343 Thanks. Its a good option
Nah, it's far away from the masterpiece our boy here made in Pycharm

**Comment by antohn** [comment]
Love this theme for many years.
Since I switched to VSC a while ago I miss the theme a lot and willing to "donate some sats" for someine to create an accurate port to VSC.
Tried fiddleing with Claude, but it does not seem to get the color right.
